apps/wallet/src/ui/app/staking/delegation-detail/DelegationDetailCard.tsx

```diff
@@ -75,7 +75,7 @@ export function DelegationDetailCard({ validatorAddress, stakedId }: DelegationD
     const { data: metadata } = useCoinMetadata(IOTA_TYPE_ARG);
     const { isCommitteeMember } = useIsValidatorCommitteeMember();
     const { isActiveValidator } = useIsActiveValidator();
-    const { isValidatorExpectedToBeInTheCommittee } =
+    const { isValidatorExpectedToBeInTheCommittee, isValidatorExpectedToBeInTheCommitteeLoading } =
```

Comment:
you assign `isLoading` to `isValidatorExpectedToBeInTheCommitteeLoading` if you do what I suggested about the rename